docs/views/avatarview.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "AvatarView"
+description: "The AvatarView control allows the user to display an avatar or the user's initials if no avatar is available."
+author: sthewissen
+ms.author: joverslu
+ms.date: 10/09/2020
+---
+
+# AvatarView
+The AvatarView control allows the user to display an avatar or the user's initials if no avatar is available. By binding the `Source` property the user can assign an image to the AvatarView. Simultaneously binding the `Text` property will allow the user to also set the initials to be shown if no valid image is provided.
+
+## Syntax
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:xct="clr-namespace:Xamarin.CommunityToolkit.Converters;assembly=Xamarin.CommunityToolkit"
+             x:Class="MyLittleApp.MainPage">
+
+     <StackLayout>
+
+        <xct:AvatarView Text="{Binding Initials}" Source="{Binding AvatarSource}" />
+
+    </StackLayout>
+
+</ContentPage>
+```
+
+## Properties
+
+|Property  |Type  |Description  |
+|---------|---------|---------|
+| BorderColor | [Color](/dotnet/api/xamarin.forms.color) | Gets or sets the border color for the AvatarView. |
```

Comment:
API doc links should use xref syntax for the following reasons:

- They are validated at build time. The format you've used here won't be validated at all.
- They are less to type.

The format is `xref:type`. For example:

```
[`Color`](xref:Xamarin.Forms.Color)
```

Reply:
Note that xrefs are case sensitive.

If in doubt, there's an xref query checker you can use: https://xref.docs.microsoft.com/query?uid=Xamarin.Forms.Color

If you get a response from the query checker, you have the correct type. If you just see [] displayed on the page, you don't have the correct type.